src/modules/package-downloader.ts

```diff
@@ -82,6 +82,6 @@ export const packageDownloader = (packageConfig: PackageConfig) => {
     return new PackageDownloader(packageConfig);
 };
 
-export const sdkDownloader = (sdkConfig: SdkConfig) => {
-    return new PackageDownloader(sdkConfig);
+export const coreDownloader = (coreConfig: CorePackageConfig) => {
```

Comment:
Is this extra function really needed? A package containing a core is just a regular package.